pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/jni/PyTorchLibrary.java

```diff
@@ -462,15 +462,15 @@ native void torchBackward(
     native long moduleLoad(
             String path, int[] device, String[] extraFileNames, String[] extraFileValues);
 
-    native long moduleLoad(InputStream is, int[] device, byte[] buffer);
+    native long moduleLoad(InputStream is, int[] device, byte[] buffer, long size);
 
     native void moduleEval(long handle);
 
     native void moduleTrain(long handle);
 
     native long moduleForward(long moduleHandle, long[] iValueHandles, boolean isTrain);
 
-    native void moduleWrite(long moduleHandle, OutputStream os, byte[] buffer);
+    native void moduleWrite(long moduleHandle, OutputStream os, byte[] buffer, boolean writeSize);
```

Comment:
We always write with size, any case need false?

Reply:
If anytime user are looking to save as PT file and use in Python